itests/hive-unit/src/main/java/org/apache/hive/jdbc/miniHS2/MiniHS2.java

```diff
@@ -222,6 +222,8 @@ private MiniHS2(HiveConf hiveConf, MiniClusterType miniClusterType, boolean useM
       // Initialize the execution engine based on cluster type
       switch (miniClusterType) {
       case TEZ:
+        // Change the engine to tez
+        hiveConf.setVar(ConfVars.HIVE_EXECUTION_ENGINE, "tez");
         // TODO: This should be making use of confDir to load configs setup for Tez, etc.
```

Comment:
LLAP mode is not default with TEZ, I think that is well understood, so not setting that should be fine.